test/web-platform-tests/to-upstream/html/semantics/forms/form-submission-0/submit-event.html

```diff
@@ -26,24 +26,25 @@
 test(() => {
   const { submitButton } = prepareForm();
   submitButton.click();
-  assert_equals(submitButton, submitEvent.submitter, "Submitter is set");
+  assert_equals(submitEvent.submitter, submitButton, "Submitter is set");
 }, "Clicking a submit button sets the submitter");
 
 test(() => {
   const { form, submitButton } = prepareForm();
   form.requestSubmit(submitButton);
-  assert_equals(submitButton, submitEvent.submitter, "Submitter is set");
+  assert_equals(submitEvent.submitter, submitButton, "Submitter is set");
 }, "Requesting submit with a submitter sets the submitter");
 
 test(() => {
   const { form } = prepareForm();
   form.requestSubmit();
-  assert_equals(undefined, submitEvent.submitter, "Submitter is not set");
+  assert_equals(submitEvent.submitter, null, "Submitter is not set");
```

Comment:
this test now fails, i'll tweak the implementation so that this property is null rather than undefined

Reply:
Done